Hyper/Scene.cs

```diff
@@ -27,21 +23,24 @@ internal class Scene : IInputSubscriber
 
     public readonly List<Humanoid> Bots = new();
 
-    public readonly List<SimpleCar> Cars;
+    public readonly List<SimpleCar> FreeCars;
```

Comment:
PlayersCar needs special treatment for Update. I can change it so that each car has a property `IsPlayerControlled` but this way it's more analogous to what we have in humanoid bots vs player